chromecastize.sh

```diff
@@ -1,5 +1,6 @@
 #!/bin/bash
 
+set -e
```

Comment:
Which kind of errors did you get that requires this flag to be set? I would prefer the script to ignore minor errors and continue running.

Since the conversion process can take many hours when converting a large number of files it would be disappointing to come back to the computer and notice that it stopped processing because of some error in one of the files.